pages/faqs.md

```diff
@@ -236,3 +236,14 @@ That’s up to you, Solid just gives you the choice.
 ## Is it wise to trust people to make ethical decisions on such a complicated issue?
 There is a lot of variation in what people feel to be right and wrong, and ultimately it is down to you to decide what to do with your data. Solid does try to help you make informed choices by providing relevant information and by constantly validating the accuracy of that information.
 
+## Are apps vulnerable to “Origin” being escaped? 
```

Comment:
I don't understand the question?

Do we mean "spoofed" or "omitted" or …?

Escaping, to me, is the process of prefixing certain characters by a dedicated character to strip those characters of any special interpretation.

Reply:
fair point. many meanings to escape. adding something is indeed a method of escape, as is omitting something. lack of special interpretation is then a bypass. in this case the bypass is authentication. i'll revise with CWE-346 text about origin validation as that's standardized language